releases/latest/kernel/helpers/build/populate_scc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -Eeox pipefail
+
+SCC_SIZE="$1" # Size of the SCC layer
+ITERATIONS="$2" # Number of iterations to run to populate it
+
+# Make sure the following Java commands don't disturb our class cache until we're ready to populate it
+# by unsetting IBM_JAVA_OPTIONS if it is currently defined.
+unset IBM_JAVA_OPTIONS
+
+# Explicity create a class cache layer for this image layer here rather than allowing
+# `server start` to do it, which will lead to problems because multiple JVMs will be started.
+java -Xshareclasses:name=liberty,cacheDir=/output/.classCache/,createLayer -Xscmx$SCC_SIZE -version
+
+# Populate the newly created class cache layer.
+export IBM_JAVA_OPTIONS="-Xshareclasses:name=liberty,cacheDir=/output/.classCache/"
```

Comment:
is this option honoured by the AdoptOpenJ9 JREs are well?   I thought there was a more generic option name.

Reply:
Also - in our Dockerfiles we have this line:

```
IBM_JAVA_OPTIONS="-Xshareclasses:name=liberty,nonfatal,cacheDir=/output/.classCache/ ${IBM_JAVA_OPTIONS}"
```

I think we will have a duplicated IBM_JAVA_OPTIONS string then?

Reply:
> is this option honoured by the AdoptOpenJ9 JREs are well? I thought there was a more generic option name.

It's still honoured for the time being. OpenJ9 builds will at some point in the future stop looking at it and only look at `OPENJ9_JAVA_OPTIONS`, but for now `IBM_JAVA_OPTIONS` still works.

Since the rest of your Dockerfiles are still using `IBM_JAVA_OPTIONS` I've decided to stick with it so that different scripts aren't using different variables for the same purpose and so you can migrate everything in one shot.

Reply:
> I think we will have a duplicated IBM_JAVA_OPTIONS string then?

The one you have on the `ENV` line will be the only one saved in the image and will take effect whenever a container is started.

The ones being set in the `RUN` lines and in the script are temporary and only exists during image build-time and won't be visible after. This is good because they contain extra options that are only being used to create and populate the SCC so image build-time is the only place they're needed.

Reply:
But in the line where we create the `ENV`, would it pick up the `IBM_JAVA_OPTIONS` that was created in the `RUN` a few lines back?

Reply:
Actually, I think it's fine. :)